entitlement/src/main/java/org/killbill/billing/entitlement/logging/EntitlementLoggingHelper.java

```diff
@@ -158,7 +159,7 @@ public static void logTransferEntitlement(final Logger log,
         }
     }
 
-    public static void logCancelEntitlement(final Logger log, final Entitlement entitlement, final LocalDate entitlementEffectiveDate, final Boolean overrideBillingEffectiveDate, final EntitlementActionPolicy entitlementPolicy, final BillingActionPolicy billingPolicy) {
+    public static void logCancelEntitlement(final Logger log, final Entitlement entitlement, final Serializable entitlementEffectiveDate, final Serializable billingEffectiveDate, final Boolean overrideBillingEffectiveDate, final EntitlementActionPolicy entitlementPolicy, final BillingActionPolicy billingPolicy) {
```

Comment:
I see.. the only common interface between `LocalDate` and `DateTime` is `Serializable` (which does not mean much) but works here as we just call `toString()`. So in this case we might as well to pass a java `Object` here which is the one defining the `toString()` ?

Reply:
Fixed